ansible_collections/arista/avd/roles/eos_validate_state/python_modules/tests/avdtestconnectivity.py

```diff
@@ -39,44 +39,66 @@ def test_definition(self) -> dict | None:
             LOGGER.warning("Variable '%s' is missing from the structured_config. %s is skipped.", str(e), self.__class__.__name__)
             return None
 
-        required_vars = ["type", "ip_address", "peer", "peer_interface"]
+        required_vars = ["name", "peer", "peer_interface", "type", "shutdown"]
 
-        for ethernet_interface in ethernet_interfaces:
+        for idx, ethernet_interface in enumerate(ethernet_interfaces, start=1):
             try:
                 for var in required_vars:
                     get(ethernet_interface, var, required=True)
+            except AristaAvdMissingVariableError as e:
+                LOGGER.warning("Ethernet interface entry #%d from the 'ethernet_interfaces' data model is missing the variable '%s'.", idx, str(e))
```

Comment:
if this patterns comes back - write a help function that is able to take a log message format. a ist of required variables and handles the logging for you

Reply:
Added in 3cb11f7